assignment2/inheritance.js

```diff
@@ -0,0 +1,17 @@
+function Employee (name) {
+    this.name = name;
+    
+                          }
+
+function Manager(salary) {
+   
+    this.salary = salary;
+    
+                          }
+
+var emp=new Employee("Samarth");
```

Comment:
space before and after an operator is expected eg: instead of `a+b` write `a + b`